zksync_sdk/zksync_provider/interface.py

```diff
@@ -34,6 +35,11 @@ async def submit_txs_batch(self, transactions: List[TransactionWithSignature],
                                ] = None) -> List[Transaction]:
         raise NotImplementedError
 
+    @abstractmethod
+    async def submit_batch_builder_trx_batch(self, transactions: List[TransactionWithOptionalSignature],
```

Comment:
Please change `submit_batch_builder_trx_batch` to `submit_batch_builder_txs_batch`

Reply:
Done